src/web/dockerflow.rs

```diff
@@ -61,3 +63,13 @@ async fn test_error(_: HttpRequest) -> Result<HttpResponse, HandlerError> {
     error!("Test Error");
     Err(HandlerError::internal("Oh Noes!"))
 }
+
+async fn document_boot(
+    _: HttpRequest,
```

Comment:
nit: `HttpRequest`'s just another extractor, might as well kill this line since it isn't used anyway (I think I removed the others in the skeleton)

Reply:
I removed the `HttpRequest` noop from the other calls.